src/TaggedInput.jsx

```diff
@@ -269,7 +269,13 @@ var TaggedInput = React.createClass({
     if (tagText && tagText.length > 0) {
       trimmedText = tagText.trim();
       if (p.unique) {
-        duplicateIndex = this.state.tags.indexOf(trimmedText);
+
+        // not a boolean, it's a function
+        if (p.unique !== true) {
```

Comment:
@MitMaro , you need to do this - if (typeof p.unique === 'function').

Without that it will cause an error if someone sets p.unique to false.